lib/internal/trace_events_async_hooks.js

```diff
@@ -70,7 +70,11 @@ function createHook() {
     enable() {
       if (this[kEnabled])
         return;
-      this[kEnabled] = true;
+      Object.defineProperty(this, kEnabled, {
+        value: true,
+        configurable: true,
+        writable: true
+      });
```

Comment:
I’d also consider this cheap and useful as debugging information.

Reply:
The same question as above: https://github.com/nodejs/node/pull/23126/files#r221182318